evm/src/executor/builder.rs

```diff
@@ -122,15 +146,14 @@ impl ExecutorBuilder {
     /// Configure the execution environment (gas limit, chain spec, ...)
     #[must_use]
     pub fn with_config(mut self, env: Env) -> Self {
-        self.inspector_config.block = env.block.clone();
```

Comment:
Something went wrong here, this is not supposed to be removed

Reply:
oops,
good catch, added back.